golang-deploy/Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM circleci/golang:1.13
+FROM cimg/go:1.18
 ARG DOCKERIZE_VERSION="v0.6.1"
 
 USER root
 RUN apt-get update && \
-    apt-get install -y npm git-crypt make sshpass awscli tree
+    apt-get install -y git-crypt make sshpass awscli tree
 RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
     sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
     rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
-RUN npm install -g serverless
+
 USER circleci
+
+RUN curl -o- -L https://slss.io/install | VERSION=2.66.1 bash
```

Comment:
this is run after `USER circleci` cause it then adds the `PATH`-instructions to the `circleci`-user-bashrc instead of the root-users bashrc